src/InnoSetup/IdfToolsSetup.iss

```diff
@@ -326,6 +327,7 @@ Name: "{#COMPONENT_TARGET_ESP32_S3}"; Description: {cm:ComponentTargetEsp32s3};
 #if OFFLINE == 'no'
 Name: "{#COMPONENT_OPTIMIZATION}"; Description: {cm:ComponentOptimization}; Types: custom;
 Name: "{#COMPONENT_OPTIMIZATION_ESPRESSIF_DOWNLOAD}"; Description: {cm:ComponentOptimizationEspressifDownload}; Types: full custom; Flags: checkablealone
+Name: "{#COMPONENT_OPTIMIZATION_JIHULAB_MIRROR}"; Description: {cm:ComponentOptimizationJihulabMirror}; Types: custom; Flags: checkablealone
```

Comment:
`But I have noticed that the Jihulab option is checked out by default, I think we should update this for the default version of the Installer to be not checked.`

@jakub-kocka I think it is already not checked. Please correct me if I was wrong

Reply:
I have had the same impression from the code but when tested on the built installer it was checked.

I will take a look at this more.

Reply:
Ok, I got it, you may currently in the [leeebo/main](https://github.com/leeebo/idf-installer/tree/main) branch which changes the default value (for my test), but the PR is from [feat/add_jihulab_git_mirror](https://github.com/leeebo/idf-installer/tree/feat/add_jihulab_git_mirror)

Reply:
You are right the `Types: custom;` is correct and then it is not checked.